html_text/tests/test_extractor.py

```diff
@@ -40,20 +40,16 @@ def test_excerpts(self):
             self.text_from_html(html, 7, ellipsis=""),
             u"I'm a title I'm a paragraph ¡Pues")
 
+    @mute_logger("openerp.addons.html_text.models.ir_fields_converter")
     def test_empty_html(self):
         """Empty HTML handled correctly."""
         self.assertEqual(self.text_from_html(""), "")
-        with self.assertRaises(etree.XMLSyntaxError):
+        with self.assertRaises(Exception):
             self.text_from_html("", fail=True)
 
+    @mute_logger("openerp.addons.html_text.models.ir_fields_converter")
     def test_false_html(self):
         """``False`` HTML handled correctly."""
         self.assertEqual(self.text_from_html(False), "")
-        with self.assertRaises(TypeError):
+        with self.assertRaises(Exception):
             self.text_from_html(False, fail=True)
-
-    def test_bad_html(self):
```

Comment:
Why removing this test?

Reply:
Because on newer LXML versions, that code is considered "good" HTML (remember that those rules are less strict than XML ones) and thus it doesn't raise an error.

Reply:
But isn't better to provide a bad HTML?

Reply:
I guess yes, but not sure on what is now considered bad exactly... I opted for this quicker solution.

Reply:
Well, we are reducing test scope. For sure multiple unclosed tags will be considered illegal.

Reply:
Nope, it seems now it eats anything:

```py
>>> html.fromstring("<<bad></p><a=neneeñ/>").xpath("//text()")
['<']
>>> html.fromstring("<<bad></p><a=neneeñ<ñ>/>").xpath("//text()")
['<', '/>']
>>> html.fromstring("<<bad></p><a=neneeñ<ñ>/>{}&al").xpath("//text()")
['<', '/>{}&al']
>>> html.fromstring("<<bad></p><a=neneeñ<ñ>/>{}&al;").xpath("//text()")
['<', '/>{}&al;']
>>> html.fromstring("<<bad></p><a=neneeñ<ñ>/>{}&x00185;").xpath("//text()")
['<', '/>{}&x00185;']
>>> html.fromstring("<<bad></p><a=neneeñ<ñ>/>{}&nbsp;").xpath("//text()")
['<', '/>{}\xa0']
```

It's a good thing, since HTML parsing should never raise exceptions, but then yes, we have to reduce the test scope now

Reply:
OK, let's accept that